StitchCore/Sources/StitchCore/Internal/Net/FoundationHTTPTransport.swift

```diff
@@ -23,13 +23,9 @@ public final class FoundationHTTPTransport: Transport {
 
         let session = URLSession(configuration: sessionConfig)
 
-        var contentHeaders = request.headers
```

Comment:
I got rid of this since the doc request builders always add this when this is necessary, and we don't want to be adding this to requests with empty bodies.